go-controller/pkg/node/gateway_shared_intf.go

```diff
@@ -288,9 +288,10 @@ func (npw *nodePortWatcher) updateServiceFlowCache(service *kapi.Service, add, h
 func (npw *nodePortWatcher) createLbAndExternalSvcFlows(service *kapi.Service, svcPort *kapi.ServicePort, add bool,
 	hasLocalHostNetworkEp bool, protocol string, actions string, externalIPOrLBIngressIPs []string, ipType string, ofPorts []string) error {
 
-	// each path has per IP generates about 4-5 flows. So we preallocate a slice with capacity = max flows * num IPs
-	externalIPFlows := make([]string, 0, 5*len(externalIPOrLBIngressIPs))
 	for _, externalIPOrLBIngressIP := range externalIPOrLBIngressIPs {
+		// each path has per IP generates about 4-5 flows. So we preallocate a slice with capacity = max flows * num IPs
+		externalIPFlows := make([]string, 0, 5*len(externalIPOrLBIngressIPs))
```

Comment:
nit: in this case we don't need to allocate it for 5*len(externalIPOrLBIngressIPs) capacity right since for each IP you are doing a new one?